Packs/GoogleCloudFunctions/Integrations/GoogleCloudFunctions/GoogleCloudFunctions_test.py

```diff
@@ -1,4 +1,16 @@
 import pytest
+from GoogleCloudFunctions import set_default_project_id
+
+
+@pytest.mark.parametrize('project, credentials_json, expected_output', [
+    ("some-project-id", {"credentials_json": {"type": "service_account", "project_id": "some-project-id"}},
+     "some-project-id"),
+    (None, {"credentials_json": {"type": "service_account", "project_id": "some-project-id"}}, "some-project-id"),
+    ("some-project-id", {"credentials_json": {"type": "service_account"}}, "some-project-id"),
+])
+def test_set_default_project_id(project, credentials_json, expected_output):
```

Comment:
Please add the edge case where you dont have both, catch the exception

Reply:
Done